src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -583,6 +583,18 @@ internal static byte[] ConvertIntToByteArray(int dwInput)
             return rgbOutput;
         }
 
+        internal static int ConvertByteArrayToInt(byte[] input)
```

Comment:
The Counter was my bad. Since the test-keys contained only such small values, I assumed, that it's only 1 byte long. Couldn't find a concrete definition after research at all.